docker/bootstrap/ubuntu-14.04/Dockerfile

```diff
@@ -48,6 +48,19 @@ RUN export DEBIAN_FRONTEND=noninteractive \
     && hash -r \
     && pip install ansible \
     && chmod 750 /usr/local/bin/ansible* \
+    # Install gosu
+    && GOSU_VERSION=1.10 \
+    && /usr/local/bin/apt-install wget gnupg \
+    && dpkgArch="amd64" \
+    && wget -O /usr/local/bin/gosu "https://github.com/tianon/gosu/releases/download/$GOSU_VERSION/gosu-$dpkgArch" \
+    && wget -O /usr/local/bin/gosu.asc "https://github.com/tianon/gosu/releases/download/$GOSU_VERSION/gosu-$dpkgArch.asc" \
```

Comment:
I've only used the part from the "official" installation and wanted to keep the steps :)
https://github.com/tianon/gosu